physionet-django/project/forms.py

```diff
@@ -750,9 +751,15 @@ class AccessMetadataForm(forms.ModelForm):
     """
     class Meta:
         model = ActiveProject
-        fields = ('access_policy', 'license')
-        help_texts = {'access_policy': '* Access policy for files.',
-                      'license': "* License for usage. <a href='/about/publish/#licenses' target='_blank'>View available.</a>"}
+        fields = ('access_policy', 'license', 'allow_file_downloads')
+        help_texts = {
+            'access_policy': '* Access policy for files.',
+            'license': "* License for usage. <a href='/about/publish/#licenses' target='_blank'>View available.</a>",
+            'allow_file_downloads': (
+                '* This option allows to enable/disable direct files downloads from the '
+                'platform. It cannot be changed after the publication of the project!'
+            ),
```

Comment:
@bemoody and I think this comment is not especially clear and it may lead to confused authors, but we are happy to leave it as it is for now and perhaps fix later.